docs/serverless-v2/03-02-api-rules.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Exposing lambdas
+type: Details
+---
+
+By default, lambdas in Kyma are not exposed outside the cluster. All Function CRs receive the `serving.knative.dev/visibility=cluster-local` label that is also automatically assigned to the KSevice CR. Without this label, the Knative Serving controller would create two virtual services, one for the internal cluster communication and the other one for exposing the lambda service outside the cluster. To restrict the access, we set the default `serving.knative.dev/visibility=cluster-local` label on KSevice CR that limits the Knative Serving controller to create only one, local virtual service that allows only for cluster-wide access to lambda services. Such a lambda is accessible for other resources within the cluster under the `{service-name}.{namespace}.svc.{domain}` endpoint, such as `test-lambda.default.svc.kyma.local`.
```

Comment:
The local address is wrong, `{domain}` is not Kyma domain, it is the internal cluster domain, by default it is `cluster.local` - default because it is possible to change that in Kubernetes configuration.

`{service-name}.{namespace}.svc.cluster.local` and `test-lambda.default.svc.cluster.local`